modules/server/outputs.tf

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-output "forseti-server-vm-name" {
-  description = "Forseti Server VM name"
-  value       = google_compute_instance.forseti-server.name
+output "forseti-cloudsql-connection-name" {
+  description = "The connection string to the CloudSQL instance"
+  value       = google_sql_database_instance.master.connection_name
 }
 
-output "forseti-server-vm-ip" {
-  description = "Forseti Server VM private IP address"
-  value       = google_compute_instance.forseti-server.network_interface[0].network_ip
+output "forseti-server-git-public-key-openssh" {
+  description = "The public OpenSSH key generated to allow the Forseti Server to clone the policy library repository."
+  value       = tls_private_key.policy_library_sync_ssh[0].public_key_openssh
```

Comment:
@gkowalski-google Have you checked that this doesn't raise any errors if policy library sync is turned off?

Reply:
Yes I have tested with the sync enabled and disabled, no issues with the actual infra setup or with the terraform outputs. In the case where it is disabled, this output is not provided.